tools/release.sh

```diff
@@ -17,6 +17,8 @@ pwd
 
 make release
 
+make dfly_bench
```

Comment:
`make release` from `cpack` does not build `dfly_bench` directly. Maybe there is a better way other than invoking make directly :shrug:

Reply:
:scream: it was a `makefile` rule :facepalm: :facepalm: